.scripts/regeneration.js

```diff
@@ -17,7 +17,7 @@ const goMappings = {
     'binarygroup': ['body-binary.json', '--remove-unreferenced-types'],
     'booleangroup': ['body-boolean.json', '--remove-unreferenced-types'],
     'bytegroup': ['body-byte.json', '--remove-unreferenced-types'],
-    'complexgroup': ['body-complex.json', '--remove-unreferenced-types'],
+    'complexgroup': ['body-complex.json', '--remove-unreferenced-types', '--rawjson-as-bytes'],
```

Comment:
This was enabled here, and a few others, where no change was expected in the generated code.